apps/web/src/pages/templates/components/ChannelPreview.tsx

```diff
@@ -0,0 +1,59 @@
+import { StepTypeEnum } from '@novu/shared';
+import { useParams } from 'react-router-dom';
+import { ChatPreview, EmailPreview, InAppPreview, PushPreview, SmsPreview } from '../../../components/workflow/preview';
+
+import { useNavigateFromEditor } from '../hooks/useNavigateFromEditor';
+import { useStepIndex } from '../hooks/useStepIndex';
+import { ChannelPreviewSidebar } from './ChannelPreviewSidebar';
+
+export const PreviewComponent = ({ channel }: { channel: StepTypeEnum }) => {
+  switch (channel) {
+    case StepTypeEnum.EMAIL:
+      return <EmailPreview view="web" showVariables={false} />;
+
+    case StepTypeEnum.TRIGGER:
+      return <>TRIGGER</>;
+
+    case StepTypeEnum.SMS:
+      return <SmsPreview />;
+
+    case StepTypeEnum.IN_APP:
+      return <InAppPreview showVariables={false} />;
+
+    case StepTypeEnum.CHAT:
+      return <ChatPreview />;
+
+    case StepTypeEnum.PUSH:
+      return <PushPreview />;
+
+    case StepTypeEnum.DELAY:
+      return <>DELAY</>;
+
+    case StepTypeEnum.DIGEST:
+      return <>DIGEST</>;
+
+    default:
+      return <>dummy</>;
```

Comment:
@ainouzgali  now that we have implemented all the channels preview, we should have some proper UI here, 
"Unexpected error occurred!" something like that

Reply:
Great points, I will do that after we deploy to dev :) I want Nikolay to be able to start testing